go-controller/pkg/node/gateway.go

```diff
@@ -472,6 +472,13 @@ func (g *gateway) SetPodNetworkAdvertised(isPodNetworkAdvertised bool) {
 	g.isPodNetworkAdvertisedLock.Lock()
 	defer g.isPodNetworkAdvertisedLock.Unlock()
 	g.isPodNetworkAdvertised = isPodNetworkAdvertised
```

Comment:
We can probably get rid of these fields and just use `g.openflowManager.defaultBridge.netConfig`